NTuplizers/test/jescJRA_cfg.py

```diff
@@ -219,4 +224,70 @@
    print 'process.source =', process.source.dumpPython()
    print 'process.maxEvents =', process.maxEvents.dumpPython()
    print 'process.options =', process.options.dumpPython()
-   print '----------------------'
+   print '----------------------------'
+
```

Comment:
the lines below are a copy-paste error introduced by me, and corrected earlier today. they should not be added.